### PR TITLE
Remove redundant elaborated type specifiers

### PR DESCRIPTION
**Pull request by tygyh**

### CODE REVIEW

File: Source/Core/Core/IOS/Network/Socket.cpp [hunk 1]
```diff
@@ -998,7 +998,7 @@ void WiiSockMan::Update()
 {
   s32 nfds = 0;
   fd_set read_fds, write_fds, except_fds;
-  struct timeval t = {0, 0};
+   timeval t = {0, 0};
```
Comment (Tilka) [inline Source/Core/Core/IOS/Network/Socket.cpp:1001]: identation

File: Source/Core/Core/IOS/USB/Bluetooth/hci.h [hunk 2]
```diff
@@ -2581,7 +2581,7 @@ struct btreq
       uint16_t btri_link_policy; /* Link Policy */
       uint16_t btri_packet_type; /* Packet Type */
     } btri;
-    struct bt_stats btrs; /* unit stats */
+     bt_stats btrs; /* unit stats */
```
Comment (Tilka) [inline Source/Core/Core/IOS/USB/Bluetooth/hci.h:2584]: indentation

File: Source/Core/Common/SDCardUtil.cpp [hunk 3]
```diff
@@ -74,7 +74,7 @@ static void WriteData(u8* out, T data)
 static unsigned int get_serial_id()
 {
   const time_t now = std::time(nullptr);
-  const struct tm tm = std::gmtime(&now)[0];
+  const tm tm = std::gmtime(&now)[0];
```
Comment (Tilka) [inline Source/Core/Common/SDCardUtil.cpp:77]: Please keep this one for readability.

**Comment by AdmiralCurtiss** [comment]
Are you sure this is right? Technically any C (not C++) struct without a typedef should be re-stated as `struct S` every time.

**Comment by JosJuice** [comment]
@tygyh Do you build these PRs before uploading them?

**Comment by tygyh** [comment]
> @tygyh Do you build these PRs before uploading them?

No, but I will from now on.

**Comment by mitaclaw** [comment]
@dolphin-emu-bot rebuild

**Pull request merged**